index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css">
+    <title>brian nyachae portfolio</title>
+  </head>
+  <body>
+   <header>
+     <nav>
+        <div class="content heading">
+          <div class="logo-container">
+            <div class="logo-item">
+              <a class="logo" href="index.html">My Logo</a>
+            </div>
+          </div>
+          <div class="menu-top">
+            <div class="menu-item">
+              &#9776;
+            </div>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section>
+        <div class="content main-section">
+          <div class="mainn">
+            <h1>I'm Brian Nyachae Glad to see you!</h1>
+          </div>
+          <div class="details">
+            <p class="description-item">
+              I'm a software developer! I can make creative UI designs, feature
+              or website. Look through my portafolio and don't hestiate to
+              contact me.
+            </p>
+          </div>
+          <section class="connect-section">
+            <h3 class="link-item">
+              <a href="#" class="connect">LET'S CONNECT</a>
+            </h3>
+          </section>
```

Comment:
- [ ] [OPTIONAL] Kindly change this `section` into a `div`.

Reply:
@aimemalaika sure thanks for the feedback